src/shared/helpers/string-type.ts

```diff
@@ -0,0 +1,12 @@
+export class StringType {
+  static isNumeric(value: string): boolean {
```

Comment:
Is there no function which does this in lodash?

Reply:
I did look at lodash - but nothing that does this.

Reply:
Okay.
A comment which describes what does the function do and return values for different inputs would be useful for future reference.

Reply:
added comment